en/docs/admin-portal/applications/types/oidc.md

```diff
@@ -0,0 +1,7 @@
+# OIDC Web Application
+
+## Overview
+Simple introduction to OIDC web applications.
```

Comment:
intro/concept info will usually be in the 'Concepts' section of the docs. Can we just provide a link to the concept from here?

Reply:
This section will contain a simple introduction to OIDC web applications. We can keep a link to concept section from here.

Reply:
+1 should be linked from concepts.

Reply:
@mevan-karu WDYT about just adding a line such as:

 "This page guides you through adding a new OIDC web application." 

the words "OIDC web application" in that sentence can link to the concept doc.